app.js

```diff
@@ -0,0 +1,58 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const addBtn = document.querySelector('.addbook');
+const bookSec = document.querySelector('.Books');
+
+class Book {
+  constructor(title, author) {
+    this.id = `_${Math.random().toString(36).substr(2, 9)}`;
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Library {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  add(title, author) {
+    const book = new Book(title, author);
+    this.books.push(book);
+    localStorage.books = JSON.stringify(this.books);
+    this.displayBooks();
+  }
+
+  remove(bookID) {
+    this.books = this.books.filter((book) => book.id !== bookID);
+    localStorage.books = JSON.stringify(this.books);
+    this.displayBooks();
+  }
+
+  displayBooks() {
+    bookSec.innerHTML = '';
+    this.books.forEach((book) => {
+      const eachBook = `<div class="book ${book.id}">
+        <h3>${book.title}</h3>
+        <h3>${book.author}</h3>
+
+        <button type="button" data-id="${book.id}" class="removebtn">Remove</button>
+        <hr>
+     </div>`;
+      bookSec.insertAdjacentHTML('beforeend', eachBook);
+      document.querySelectorAll('.removebtn').forEach((btn) => {
+        btn.addEventListener('click', () => {
+          this.remove(btn.dataset.id);
+        });
+      });
+    });
+  }
+}
```

Comment:
- [ ] I think this method is buggy. When I create some books it gets stored in the local storage and displayed in the DOM properly. But, when I refresh the page, the local storage data persists but the list of books disappears from the DOM. Kindly fix this issue! :pray: 

![Screenshot from 2021-11-25 11-17-51](https://user-images.githubusercontent.com/64894927/143405196-3844af7c-0d0d-4c8b-9aef-25e0ac36c84c.png)

Reply:
Quick question, when I reload, the localStorage should be cleaned as well/ or the list of books shouldn't dissapear?

Reply:
Hi @BonkeG, thank you for asking. The list of books shouldn't disappear. The localstorage is completely ok.